selftest/struct-ops/.gitignore

```diff
@@ -0,0 +1 @@
+scx
```

Comment:
Would you mind moving scx folder into selftest/common, so it can be used for next sched-ext tests.